README.md

```diff
@@ -97,178 +91,148 @@
   <p>To download Ignition, please visit the Induction Automation downloads webpage in your web browser at <a href="https://inductiveautomation.com/downloads" class="uri">https://inductiveautomation.com/downloads</a>.</p>
   <p>For information on how to install Ignition and access Ignition’s web page, please visit the Ignition quick start guide at <a href="https://docs.inductiveautomation.com/display/DOC80/Quick+Start+Guide" class="uri">https://docs.inductiveautomation.com/display/DOC80/Quick+Start+Guide</a>.</p>
   <p><strong>NOTE:</strong> The Ignition Quick Start Guide contains various sections, some of which are not related to the Ewon Ignition Demo. For the Ewon Ignition Demo, it is only necessary to follow the instructions and information found in section 1, “Download and Install Ignition” and section 2, “Launch the Gateway Webpage.” During Ignition installation, you may choose “Typical” installation.</p>
   <br />
-
   <!-- APPENDICES/RESTARTING IGNITION -->
   <h3 id="restarting-ignition"><span style="font-variant: small-caps;">Restarting Ignition</span></h3>
   <p>Ignition can be restarted using two methods, listed below.</p>
   <p><strong>Method 1 (Easiest):</strong> Restart your PC.</p>
   <p><strong>Method 2 (Advanced)</strong>: On Microsoft Windows operating systems, browse to <code>C:\Program Files\Inductive Automation\Ignition</code>, and run the <code>stop-ignition.bat</code> script to stop the Ignition gateway process. Once the <code>stop-ignition.bat</code> script finishes running and closes, run the <code>start-ignition.bat</code> script to start the Ignition gateway process again. On Linux operating systems, browse to <code>/usr/local/bin/ignition</code> and run the <code>ignition.sh</code> script with the restart parameter, <code>./ignition.sh restart</code></p>
   <p><strong>NOTE:</strong> If you customized the Ignition installation directory or are using an operating system not listed, your restart procedure may be different.</p>
   <br />
-
   <!-- APPENDICES/OBTAINING EWON NAME -->
   <h3 id="obtaining-an-ewons-name"><span style="font-variant: small-caps;">Obtaining an Ewon’s Name</span></h3>
-  <p>To obtain an Ewon device’s name, you’ll need to access the Ewon device’s webpage. Navigate to the Ewon identification information page by selecting <code>Setup</code> &gt; <code>System</code> &gt; <code>Main</code> &gt; <code>General</code> &gt; <code>Identification</code>. Your Ewon’s name displays under <code>eWON Identification</code>.</p>
-</body>
-
-</html>
+  <p>To obtain an Ewon device’s name, you’ll need to access the Ewon device’s webpage. Navigate to the Ewon identification information page by selecting <code>Setup</code> &gt; <code>System</code> &gt; <code>Main</code> &gt; <code>General</code> &gt; <code>Identification</code>. Your Ewon’s name displays under <code>eWON Identification</code>.</p>
```

Comment:
maybe do a commit for changing tags with a small blurb in the commit message about that, then one for formatting changes, and the last as what's in here.